src/loader/loader.ts

```diff
@@ -37,7 +36,19 @@ export class Loader {
     }
 
     showReload(): void {
-        document.getElementById('workspace-loader-reload')!.style.display = 'block';
+        const reloadEl = document.getElementById('workspace-loader-reload');
+        if (reloadEl) {
+            const {pathname, search} = window.location;
+            const isDebugMode = search.includes(DEBUG_PARAM);
+            let href: string;
+            if (search === '') {
+                href = `${pathname}?${DEBUG_PARAM}`;
+            } else {
+                href = `${pathname}${search}${!isDebugMode ? `&${DEBUG_PARAM}` : ''}`;
```

Comment:
wondering how this should work... I would avoid using string interpolation inside ternary operator inside string interpolation.

Reply:
It works perfectly. I tested. But I agree to simplify.
```
                if (isDebugMode) {
                    href = `${pathname}${search}`;
                } else {
                    href = `${pathname}${search}&${DEBUG_PARAM}`;
                }
```
Is it OK solution for you?

Reply:
Done